MaterialDesignThemes.Wpf/Behaviors/PasswordBoxBehavior.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Xaml.Behaviors;
+
+namespace MaterialDesignThemes.Wpf.Behaviors
```

Comment:
I would use a file-scoped namespace here
```suggestion
namespace MaterialDesignThemes.Wpf.Behaviors;
```

Reply:
Changed to file-scoped namespace.